src/Widgets/Proxy/ProxyConfigurationPage.vala

```diff
@@ -34,9 +34,10 @@ namespace Network.Widgets {
         private Gtk.SpinButton ftp_spin;
         private Gtk.SpinButton socks_spin;
 
-        private Gtk.Button apply_button;
+        private Gtk.Switch *enable_sw;
```

Comment:
Please don't use the C specific syntax here. Vala already has everything as a reference and you don't need that here. You could also change this variable name to something like `root_switch`.
Just `private Gtk.Switch root_switch;`.